doc/getting-started-guide/quick-overview.rst

```diff
@@ -8,7 +8,7 @@ documentation.
 
 To begin, import numpy, pandas and xarray using their customary abbreviations:
 
-.. ipython:: python
+.. jupyter-execute::
```

Comment:
It was really quite straightforward to do this, and the resulting quickstart page in my opinion looks a lot more inviting for new users! I'd be happy to fully convert the other docs in a separate PR if people like it.

Reply:
https://xray--8708.org.readthedocs.build/en/8708/getting-started-guide/quick-overview.html 
vs 
https://docs.xarray.dev/en/stable/getting-started-guide/quick-overview.html

note: except if 'dark mode' on the theme is set, I opened https://github.com/pydata/pydata-sphinx-theme/issues/2189 to sort that out